### PR TITLE
Intercept Nametags

### PR DESCRIPTION
**Pull request by esteldunedain**
WIP

### CODE REVIEW

File: extensions/nametags/ace_nametags.cpp [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+#include "intercept.hpp"
+#include "logging.hpp"
+#include "client\client.hpp"
+#include "client\pointers.hpp"
+#include "client\sqf\uncategorized.hpp"
+#include "client\sqf\config.hpp"
+
+#include "nametagger.hpp"
+
+#define PI 3.14159265f
```
Comment (bux578) [inline extensions/nametags/ace_nametags.cpp]: `include <math.h>`
`M_PI`
?

Comment (esteldunedain) [inline extensions/nametags/ace_nametags.cpp]: Sure. But I think I'm not even using PI or any of the math functions. It's just copy/pasted crap.

**Comment by dedmen** [comment]
I worte an updated version to latest Intercept version at https://github.com/dedmen/intercept_ace_nametags
It's just a quick mockup.

**Comment by bux** [comment]
Do we need to keep this open?

**Comment by dedmen** [comment]
Is there still Interest in Intercept Nametags/https://github.com/acemod/ACE3/pull/3222?
If yes I would rebuild it.
The old Intercept PR's are very outdated and would need a complete rewrite.

**Comment by jonpas** [comment]
Then I say close when new one is open.

I think there is still interest, I am all for starting to use Intercept, as long as we can do it optionally and not force something that might not work or be unstable (especially since Intercept is not 1.0 yet), it will help greatly with Intercept development and awareness IMO.

**Pull request closed without merging**